README-CHANGES.xml

```diff
@@ -273,6 +273,9 @@
     <c:release date="2023-03-14T23:06:06+00:00" is-open="true" ticket-system="org.nypl.jira" version="1.5.0">
       <c:changes/>
     </c:release>
+    <c:release date="2023-03-14T23:06:06+00:00" is-open="true" ticket-system="org.nypl.jira" version="1.5.1">
+        <c:change date="2023-03-30T00:00:00+00:00" summary="Save reading position of audiobooks on play, pause, and stop (which also includes when seeking and changing chapters)."/>
+    </c:release>
```

Comment:
I think you've made the same mistake on this PR as you did [here](https://github.com/ThePalaceProject/android-audiobook/pull/58), because this is one version ahead of the current version